### PR TITLE
Ajaxified Adding Feeds. Put some validity checks around uploaded urls. W...

### PR DESCRIPTION
**Pull request by npetryk**
...ill redraw iframe only when a new feed is added

### CODE REVIEW

**Comment by npetryk** [comment]
Left some debugging stuff in there

File: app/controllers/RSSFeeds.java [hunk 1]
```diff
@@ -15,14 +17,18 @@
...
 			User u = user();
-			RSSFeed feed = new RSSFeed(u, url);
-			// Add Unique Constraint Check
-			feed.save();
+			RSSFeed feed = new RSSFeed(u, url.trim());
+      if (feed.is_valid) {
+        try {
+          feed.save();
+          renderJSON("1");
+        } catch (ConstraintViolationException e) {
```
Comment (amshali) [inline app/controllers/RSSFeeds.java:27]: What will you do on this exception? How come it is not handled?

File: app/controllers/RSSFeeds.java [hunk 2]
```diff
@@ -15,14 +17,18 @@
...
+        }
+      }
 		}
-
-		redirect("/");
+    renderJSON("0");
```
Comment (amshali) [inline app/controllers/RSSFeeds.java:31]: What is 0?

File: app/controllers/RSSFeeds.java [hunk 3]
```diff
@@ -32,9 +38,11 @@ public static void RSSfeeds(Long userId) {
 
 
 			for(RSSFeed f : current.feeds) {
+        System.out.println(f);
```
Comment (amshali) [inline app/controllers/RSSFeeds.java:41]: Remove the systemout.

File: app/controllers/RSSFeeds.java [hunk 4]
```diff
@@ -32,9 +38,11 @@ public static void RSSfeeds(Long userId) {
...
+        if (!f.is_valid) continue;
 				feeds.add(f);
 			}
-
+      System.out.println(feeds);
```
Comment (amshali) [inline app/controllers/RSSFeeds.java:45]: If this is for debugging purposes, please remove it.

File: app/views/tags/rhsNav.html [hunk 5]
```diff
@@ -3,12 +3,26 @@
...
+    #{form action:@RSSFeeds.addFeed(), method:'GET', style:'display:inline', id:'RSSAddForm'}
+      <input type='text' name='url' style="width:100%"></input>
       <input type='submit' value='Add Feed URL'></input>
     #{/form}
+    <script type="text/javascript">
+      $('#RSSAddForm').submit(function() {
+        var form = $(this);
+        $.getJSON(form.attr('action'), form.serialize(), function(data){
+            if (data=='0') {
+              alert("'"+form.find('input[type="text"]').val()+"' is not a valid RSS url!");
```
Comment (amshali) [inline app/views/tags/rhsNav.html:17]: What does the alert do?

**Pull request closed without merging**